carbon/lib/constants.ts

```diff
@@ -42,6 +42,9 @@ const api_url =
     : // Client side we use the proxy
       "/api";
 
+export const CACHE_DURATION = 3600;
+export const TOKENS_CACHE_DURATION = 60;
```

Comment:
```suggestion
export const CACHE_DURATION_SECONDS = 3600;
export const TOKENS_CACHE_DURATION_SECONDS = 60;
```

Please add units here